src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/ScoreSystem.java

```diff
@@ -110,11 +111,25 @@ private void checkAndResetGameOnScore(ActivateEvent event, EntityRef entity) {
             EntityRef heldFlag = getHeldFlag(player);
             if (checkIfTeamScores(baseTeamComponent, heldFlag)) {
                 incrementScore(baseTeamComponent);
-                if (redScore < LASUtils.GOAL_SCORE && blackScore < LASUtils.GOAL_SCORE) {
-                    resetRound(baseTeamComponent, heldFlag);
-                } else {
-                    resetLevel(player, baseTeamComponent, heldFlag);
+                resetRound(baseTeamComponent, heldFlag);
+                if (redScore == LASUtils.GOAL_SCORE) {
+                    logger.info("sending red won");
```

Comment:
When ready to consider the PR final you might want to either remove or set debug logging to the debug level :-)